admin/src/components/ProposalDetail/index.tsx

```diff
@@ -42,12 +54,48 @@ class ProposalDetailNaked extends React.Component<Props, State> {
         okText="delete"
         cancelText="cancel"
       >
-        <Button icon="delete" block>
+        <Button icon="delete" className="ProposalDetail-controls-control" block>
           Delete
         </Button>
       </Popconfirm>
     );
 
+    const renderMatching = () => (
+      <Popconfirm
+        overlayClassName="ProposalDetail-popover-overlay"
+        onConfirm={this.handleToggleMatching}
+        title={
+          <>
+            <div>Turn {p.contributionMatching ? 'off' : 'on'} contribution matching?</div>
+            {p.status === PROPOSAL_STATUS.LIVE && (
+              <div>
+                This is a LIVE proposal, this will alter the funding state of the
```

Comment:
Interesting! Trying to think this through. What happens if a proposal is > 50% funded before matching is turned on?

Reply:
It will appear fully funded on frontend, of course the amount being matched will be < 50%, which seems like it would be ok provided we give a nice accounting to admins.

Reply:
The worst case I can think of is turning off matching on a proposal @ > 50% funding. Users would have received funded notifications and be under the impression that they can start work and request payouts.